src/ROUTES.js

```diff
@@ -13,6 +13,8 @@ export default {
     SETTINGS_PROFILE: 'settings/profile',
     SETTINGS_PREFERENCES: 'settings/preferences',
     SETTINGS_PASSWORD: 'settings/password',
+    SETTINGS_ABOUT: 'settings/about',
+    SETTINGS_APP_DOWNLOAD_LINKS: 'settings/about/appDownloadLinks',
```

Comment:
Incorrect URL naming. Should be lowercase & we can use hyphens to separate words. like `settings/about/app-download-links`. @Julesssss